package.json

```diff
@@ -34,6 +34,7 @@
     "commander": "^2.16.0",
     "directory-exists": "^2.0.0",
     "ejs": "^2.6.1",
+    "fs": "0.0.1-security",
     "lodash": "^4.17.10",
```

Comment:
@scorsi everything looks good with this pull request. Why do we need to explicitly add the **fs** module?